### ISSUE TITLE
Could you expose methods for the host to play via API? :)

### ISSUE THREAD

**Issue by Cerebrock** [haxball/haxball-issues#486]
Would love to implement a reinforcement learning bot for academic purposes

**Comment by olegmisar** [comment]
U can't move player via API. I tried to get around of this problem by using python websocket server but I am not good at AI yet.

**Comment by tohey** [comment]
@olegmisar Have you actually made this work? I tried similar things too but I wasn't able to dispatch events via JS console. That's why I still have fingers crossed for basro to implement this on the headless API. If there are working examples, I would love to see them. We should at least be able to implement our own goalkeepers :)

**Comment by olegmisar** [comment]
Yes but I found simpler method. You can use `open()` function. Something like this should work:
```js
let gameWindow = open('https://www.haxball.com/play')
gameWindow.dispatchEvent(new Event())
```

**Comment by olegmisar** [comment]
Make sure that you use the same domain for headless host and `open()`. Note that www.haxball.com and html5.haxball.com are different domains.
